README.md

```diff
@@ -215,7 +215,7 @@ app.use exprestive
   * default: `__dirname`
 * `controllersPattern`
   * [Glob](https://github.com/isaacs/node-glob) pattern used to find controllers. Resolved relative to `appDir`
-  * default: `'controllers/*_controller.{coffee,js}'`
+  * default: `'controllers/*_controller.{coffee,js,ls}'`
```

Comment:
Capital letters in file names for required files are a code smell. It means your codebase may break if moving from a case sensitive file system to a case insensitive one.